.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - actions/checkout@v2.3.4
+      - actions/setup-node@v2.1.2
+      - yarn install --frozen-lockfile
+      - yarn run lint
+
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - "15"
+          - "14"
+          - "10"
+        angular:
+          - ""
+          - "^9.1.0"
+          - "~10.0.0"
+    name: Node.js ${{ matrix.node }} / angular ${{ matrix.angular }}
+    runs-on: ubuntu-latest
+    env:
+      - ENABLE_COVERAGE: 1
```

Ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idenv

`env` should be a map:

```suggestion
      ENABLE_COVERAGE: 1
```